packages/core/src/__tests__/rum/instrumentation/DdRumUserInteractionTracking.test.tsx

```diff
@@ -185,4 +185,130 @@ it('M intercept and send a RUM event W onPress { TouchableWithoutFeedback compon
+        rendersCount++;
+        return (
+            <TouchableWithoutFeedback
+                onPress={props.onPress}>
+                <View style={styles.button}>
+                    <Text style={styles.button}>Click me</Text>
+                </View>
+            </TouchableWithoutFeedback>
+        )
+    }
+    const stableOnPress = () => {};
+    const MemoizedComponent = React.memo(DummyComponent);
+    const { rerender } = render(<MemoizedComponent onPress={stableOnPress} />);
+    expect(rendersCount).toBe(1);
+
+    // WHEN
+    rerender(<MemoizedComponent onPress={stableOnPress} />);
+    // THEN
+    expect(rendersCount).toBe(1);
+
+    // WHEN
+    rerender(<MemoizedComponent onPress={() => {}} />);
+    // THEN
+    expect(rendersCount).toBe(2);
+})
+
+it('M keeps memoization working for elements W no onPress prop is passed', async () => {
+    // GIVEN
+    DdRumUserInteractionTracking.startTracking()
+    let rendersCount = 0;
+    const DummyComponent = (props) => {
+        rendersCount++;
+        return (
+            <View style={styles.button}>
+                <Text style={styles.button}>{props.title}</Text>
+            </View>
+        )
+    }
+    const MemoizedComponent = React.memo(DummyComponent);
+    const { rerender } = render(<MemoizedComponent title={'Click me'} />);
+    expect(rendersCount).toBe(1);
+
+    // WHEN
+    rerender(<MemoizedComponent title={'Click me'} />);
+    // THEN
+    expect(rendersCount).toBe(1);
+
+    // WHEN
+    rerender(<MemoizedComponent title={'New title'} />);
+    // THEN
+    expect(rendersCount).toBe(2);
+})
+
+it('M keeps memoization working for elements W an onPress prop is passed and custom arePropsEqual specified', async () => {
+    // GIVEN
+    DdRumUserInteractionTracking.startTracking()
+    let rendersCount = 0;
+    const DummyComponent = (props) => {
+        rendersCount++;
+        return (
+            <TouchableWithoutFeedback
+                onPress={props.onPress}>
+                <View style={styles.button}>
+                    <Text style={styles.button}>{props.title}</Text>
+                </View>
+            </TouchableWithoutFeedback>
+        )
+    }
+    const stableOnPress = () => {};
+    const MemoizedComponent = React.memo(
+        DummyComponent,
+        (previousProps, nextProps) => previousProps.title === nextProps.title
+    );
+    const { rerender } = render(<MemoizedComponent onPress={stableOnPress} title={'Click me'} />);
+    expect(rendersCount).toBe(1);
+
+    // WHEN
+    rerender(<MemoizedComponent onPress={stableOnPress} title={'Click me'} />);
+    // THEN
+    expect(rendersCount).toBe(1);
+
+    // WHEN
+    rerender(<MemoizedComponent onPress={stableOnPress} title={'New title'} />);
+    // THEN
+    expect(rendersCount).toBe(2);
+})
+
+it('M keeps memoization working for elements W an onPress prop is passed and custom arePropsEqual specified including onPress check', async () => {
```

Comment:
this line looks a bit long (also the rule for the future formatter).

Reply:
I agree. Ok for you if we leave all the linting issues aside for now so they are all properly fixed once we put new rules in place?

Reply:
yes, definitely, it is just a comment for the future improvements.